plenty_api/api.py

```diff
@@ -1308,6 +1310,17 @@ def plenty_api_get_shipping_packages_for_order(self, order_id: int,
         return utils.summarize_shipment_packages(
             response=package_responses, mode=mode)
 
+    def plenty_api_get_amazon_product_types(self):
+        """
+        Get a list of all available Amazon product types.
+        https://developers.plentymarkets.com/en-gb/plentymarkets-rest-api/index.html#/Pim/get_rest_pim_amazon_product_types
+        Return:
+                        [JSON(Dict) / DataFrame] <= self.data_format
+        """
+        domain = 'pim' # pim is the REST Route not the domain
+        path = '/amazon-product-types' # path in this case is the endpoint
```

Comment:
Also, please enter this path directly into the call, this string is quite short.

E.g.
```
        return self.__plenty_api_generic_get(domain='pim',
                                             path='/amazon-product-types')
```

Reply:
same for here, but sure